src/dune_rules/dune_file.mli

```diff
@@ -266,6 +266,8 @@ module Executables : sig
     ; forbidden_libraries : (Loc.t * Lib_name.t) list
     ; bootstrap_info : string option
     ; enabled_if : Blang.t
+    ; sub_systems : Sub_system_info.t Sub_system_name.Map.t
```

Comment:
I'm a bit surprised that you need this. Perhaps you could explain what your plan is and we could use if a sub system is really what you need.

Reply:
I don't need `sub_systems` for anything directly, but I am constructing library stanzas elsewhere from an `Executables.t` `Buildable.t` if it defines `ctypes`, and, unfortunately, those have non-optional `sub_systems` fields.  So that seems like the easiest place to get at it.

Is there a sensible value I could bind at `Sub_systems_info.default`?

Reply:
What about just using `Sub_system_name.Map.empty` when constructing the library?